tasks/configure.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: Copy new td-agent-bit.conf from role's default
+  template:
+    src: td-agent-bit.conf.j2
+    dest: "/etc/td-agent-bit/td-agent-bit.conf"
+  changed_when: false
```

Comment:
Why `changed_when` is used with template module?

Reply:
to pass indepotence test

Reply:
template module is idempotent by default.